### DISCUSSION TITLE
Import / Sync wishlist from Steam

### DISCUSSION THREAD

**Discussion by thesunnyk** [AppOutlet/GameOutlet#154] [Ideas]
- Many people already have a substantial Steam wishlist
- It's often important for game companies that their game be wishlisted on Steam. 
- It's convenient to wishlist a game straight from Steam's interface.

It would be nice, therefore, to sync or import wishlists from Steam

**Comment by MessiasLima** [comment]
Thank you for the suggestion. I will be looking into it soon
